scripts/flamingfir-deploy.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+set -x
+
+TARGET_HOST="$1"
+COMMIT=$(echo ${CI_BUILD_REF} | cut -c -9)
+DOWNLOAD_URL="https://releases.parity.io/substrate/x86_64-debian:stretch/2.0.0-${COMMIT}/substrate"
```

Comment:
the short hash can have a variable width that's why I put https://github.com/paritytech/substrate/pull/2599/files#diff-96edf7a6f008de9e928d04e1ae5e12a5R148 matching `{7,13}` and use the latest tag later on.

Reply:
So switch to the sed statement??

Reply:
Do you intend to deploy other versions than the ones that is build in the current pipeline? If not then the VERSION file could be used or just the `latest` tag in the path.

Reply:
(artifacts/VERSION) file

Reply:
No, Andre mentioned this is master pipelines only...

Reply:
@gabreal I would want to avoid using ```latest``` in the command in case there are multiple jobs running at once. The job provides the CI_BUILD_REF so is there any major reason not to use a part of that?

Reply:
Well, the fact that I'm matching `{7,13}` was a result in the length mismatch of a short hash. the short hash that is used for the urls derives from the output of `substrate --version` which can be can be different from the gitlab variables. And its length was also not constant. That's why I think it makes sense either to change the whole ci to use the variable - I thought it could cause confusion because of the output - or to query it in the script, or use the file `artifacts/VERSION`, or use the latest tag (I think a race condition is a valid concern for the master branch).